File: frontend/src/models/bid.ts
export interface Bid {
  bidID: number;
  itemID: number;
  userID: number;
  askingPrice: number;
  status: string;
  published: string;
  itemName: string;
  email: string;
}

export interface PlaceBid {
  itemID: number;
  askingPrice: number;
}

File: backend/src/main/java/edu/ntnu/idatt2105/backend/model/Bid.java
package edu.ntnu.idatt2105.backend.model;

import lombok.Getter;
import lombok.Setter;

/**
 * A model representing a bid.
 */
@Getter
@Setter
public class Bid {
  private int bidID;
  private int itemID;
  private int userID;
  private int askingPrice;
  private String status;
  private String published;
  private String itemName;
  private String email;
}
